x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -327,9 +328,27 @@ export function XyToolbar(props: VisualizationToolbarProps<State>) {
               {isFittingEnabled ? (
                 <EuiFormRow
                   display="columnCompressed"
-                  label={i18n.translate('xpack.lens.xyChart.missingValuesLabel', {
-                    defaultMessage: 'Missing values',
-                  })}
+                  label={
+                    <>
+                      {i18n.translate('xpack.lens.xyChart.missingValuesLabel', {
+                        defaultMessage: 'Missing values',
+                      })}{' '}
+                      <EuiIconTip
+                        color="subdued"
+                        content={i18n.translate('xpack.lens.xyChart.missingValuesLabelHelpText', {
+                          defaultMessage: `Lens provides a set of fitting functions to apply specific behaviour if 
```

Comment:
I recommend starting this with `When there are missing values in an area or line chart...`

I'm unsure about what the second sentence means. If the missing values are represented by dotted lines, then how can they fill other missing values?

Reply:
The second sentence tries to explain the following:
* missing values before data => ignored
* missing values within two values => represented with dotted lines (the specific curve used depends on the option selected)